plugins/cisco_umbrella_destinations/setup.py

```diff
@@ -0,0 +1,14 @@
+# GENERATED BY KOMAND SDK - DO NOT EDIT
+from setuptools import setup, find_packages
+
+
+setup(name="cisco_umbrella_destinations-conor-plugin",
+      version="1.0.3",
+      description="Cisco Umbrella Destinations allows users to manage, block, and allow network destinations based on policies within their organization",
+      author="conor",
```

Comment:
```suggestion
      author="rapid7",
```

Reply:
This was only changed so I could upload to dev, thanks for noticing!